build/gulpfile.vscode.js

```diff
@@ -242,6 +249,26 @@ function computeChecksum(filename) {
 	return hash;
 }
 
+function packageBuiltInExtensions() {
+	const sqlBuiltInlocalExtensionDescriptions = glob.sync('extensions/*/package.json')
```

Comment:
`sqlBuiltInlocalExtensionDescriptions` should probably be `sqlBuiltInLocalExtensionDescriptions`  with capital `L` in `local`.